### PR TITLE
Hmda page

### PR DESCRIPTION
**Pull request by schbetsy**
[Short description explaining the high-level reason for the pull request]

## Additions

-

## Removals

-

## Changes

-

## Testing

1.

## Notes

-

## Todos

-

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome on desktop
- [ ] Firefox
- [ ] Safari on macOS
- [ ] Edge
- [ ] Internet Explorer 9, 10, and 11
- [ ] Safari on iOS
- [ ] Chrome on Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

**Comment by chosak** [comment]
Our Python isort config currently requires all of our apps to be explicitly listed [here](https://github.com/cfpb/cfgov-refresh/blob/6fadca2a962dfd1fcf0a8e180c0d672673b9a4fd/.isort.cfg#L12). This new `hmda` app will be needed to added to the list.

File: cfgov/hmda/models/forms.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+            'multiple': 'multiple',
+        })
+    )
+
+    action = forms.ChoiceField(
+        required=False,
+        choices=HMDA_ACTIONS,
+        widget=widgets.RadioSelect(attrs={
+            'id': 'o-hmda_actions',
+            'class': 'o-hmda_actions',
```
Comment (anselmbradford) [inline cfgov/hmda/models/forms.py]: `o-hmda` is definitely not an organism. This likely should be `m-form-field m-form-field__radio` etc from https://cfpb.github.io/capital-framework/components/cf-forms/

File: cfgov/hmda/jinja2/hmda/hmda-explorer-controls.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+   value: Data object from an instance of the FilterableList StreamField block.
+
+   form: Django form that carries the fields that are to be rendered.
+
+   fragment_id: A unique ID for the filter list control.
+
+   ========================================================================== #}
+{% macro render(value, form, fragment_id, has_active_filters) %}
+    {% from 'organisms/expandable.html' import expandable with context %}
+    <div id="{{ fragment_id }}" class="o-filterable-list-controls u-mb30">
```
Comment (anselmbradford) [inline cfgov/hmda/jinja2/hmda/hmda-explorer-controls.html]: Do we need the `u-mb30`?

File: cfgov/hmda/jinja2/hmda/hmda-explorer-results.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+   years: Years of HMDA data to show
+
+   states: U.S. states of HMDA data to show
+
+   action: Mortgage action taken
+
+   ========================================================================== #}
+{% macro render(years, states, action) %}
+    <div class="u-mb30">
```
Comment (anselmbradford) [inline cfgov/hmda/jinja2/hmda/hmda-explorer-results.html]: Very likely should be a `block block__flush-top`… buuuuuuut the container for a component should appear outside the component, so this should probably wrap where ever this is called.

**Comment by schbetsy** [comment]
Closing in favor of a shared branch. Will PR soon.

**Pull request closed without merging**